src/pages/organize/[orgId]/settings/index.tsx

```diff
@@ -24,6 +25,7 @@ export const getServerSideProps: GetServerSideProps = scaffold(
   },
   {
     authLevelRequired: 2,
+    featuresRequired: [OFFICIALS],
```

Comment:
Let's remove this. You've done the right thing to add it, but we actually need this page to be accessible by those who know the URL. It just shouldn't be accessed accidentally.

Please verify that if we remove this change, the page can be accessed, just not listed in the menu.